app/v2/errors.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, sending_limit):
 
 class TooManySMSRequestsError(InvalidRequest):
     status_code = 429
-    message_template = "Exceeded sms send limits ({}) for today"
+    message_template = "Exceeded SMS daily sending limit of {}"
```

Comment:
I think if we leave the number in (which is fine because you've removed the confusion by explicitly said what it is) then we should say its for fragments, maybe something like "Exceeded SMS fragments daily sending limit of {}"?